docs/qupath.rst

```diff
@@ -1,69 +1,147 @@
+
+**Resources:**
+--------------
+
+- Data: example images from
+
+  - IDR data http://idr.openmicroscopy.org/webclient/?show=project-101
+
+-  Plugin ``ome-omero-roitool`` for import and export of ROIs to or from OMERO using OME-XML format
+
+   - https://github.com/glencoesoftware/ome-omero-roitool
+
+-  Scripts for export and import of ROIs from/to QuPath in OME-XML format
+
+   - https://github.com/glencoesoftware/ome-omero-roitool/tree/master/src/dist/QuPath.scripts
+
+
+Setup:
+------
+
+Download QuPath v0.2.0 or later from https://qupath.github.io/.
+
+
+**Step-by-step:**
+-----------------
+
+#. In OMERO.web, identify an image in the idr0018 project and the dataset Baz1a-14-100-gastrointestinal.
+
+#. Select the first image and double-click on it. This will open the image in OMERO.iviewer.
+
+#. In the OMERO.iviewer tab, select the whole URL in the address bar of your browser and copy it, for example using right-click and Copy.
+
+#. In QuPath, select *File > Open URL...* and paste the link into the dialog.
+
+   |image0|
+
+#. Click *OK*. In the following dialog, enter your credentials.
+
+   |image1|
+
+#. Set image type to *Brightfield H&E* in the following dialog.
+
+#. Find a region with well-defined cells and nuclei in the image, zoom in.
+
+#. Draw an ROI Annotation which denotes the region in which the cells will be detected using the *Wand* tool |image2|.
+
+#. Adjust your ROI Annotation using the *Brush* tool |image3|.
+
+#. Select *Analyze > Cell analysis > Cell* detection.
+
+#. You can adjust the parameters. Click *Run*. This will draw red ROIs around cells and nuclei inside your ROI Annotation.
+
+   |image4|
+
+#. Select *Measure > Show detection measurements*.
+
+   |image5|
+
+#. Note: You can save the results locally by clicking *Save* in the bottom right of the *Detection results table*.
+
+#. In the following steps, we will show how to convert the ROIs your just created in QuPath into OMERO ROIs and attach them to the image in OMERO.
+
+#. First, use the ROI OME-XML export script to export your ROIs from QuPath into OME-XML file. Find the export script in https://github.com/glencoesoftware/ome-omero-roitool/tree/master/src/dist/QuPath.scripts
+
+#. Copy the script text.
+
+#. In QuPath, open *Automate > Show script editor*
+
+#. Follow the instructions from step 3. onwards in https://github.com/glencoesoftware/ome-omero-roitool#export-ome-xml-rois to execute the export script in QuPath. This will produce a local OME-XML file.
 
-#.  Select the first image and double-click on it. This will open the
-    image in OMERO.iviewer.
+#. Note: If you run a *Cell detection* in QuPath, the nuclei ROIs will be drawn as well as the ROIs around the cells. The ROI OME-XML export script will export both the ROIs around the cells as well as the nuclei ROIs. In OMERO, they will appear as two Shapes inside the ROI belonging to the particular cell.
```

Comment:
cf. comment in slack channel. @melissalinkert is working on a fix for that

Reply:
I suppose this is a fix of a feature which is not described here (``cell detection workflow``).. All the steps described here work exactly as written.